### ISSUE TITLE
Add a move operation where rotation centers are moved as well

### ISSUE THREAD

**Issue by louisrousseau** [ijprest/keyboard-layout-editor#158]
I'd like it to be possible to move keys and have their rotation center follow as well as a toggle or option of some sort.

**Comment by ReneSac** [comment]
Agreed. Plus a bulk rotate option that also adjusts the keys rotation center so they move together! Ok, this is starting to become confusing...
